lib/remoteproc/rsc_table_parser.c

```diff
@@ -77,15 +77,14 @@ int handle_rsc_table(struct remoteproc *rproc,
 }
 
 /**
- * handle_carve_out_rsc
+ * @internal
  *
- * Carveout resource handler.
+ * @brief Carveout resource handler.
  *
- * @param rproc - pointer to remote remoteproc
- * @param rsc   - pointer to carveout resource
- *
- * @returns - 0 for success, or negative value for failure
+ * @param rproc	Pointer to remote remoteproc
+ * @param rsc	Pointer to carveout resource
  *
+ * @return 0 for success, or negative value for failure
```

Comment:
could you move this in the header file to be consistent  with the rest of the code